pkg/cmd/ide.go

```diff
@@ -26,7 +26,12 @@ var ideCmd = &cobra.Command{
 		ideList := config.GetIdeList()
 		var chosenIde config.Ide
 
-		chosenIdeId := ide.GetIdeIdFromPrompt(ideList)
+		activeProfile, err := c.GetActiveProfile()
```

Comment:
I wouldn't add the profile footer here as the IDE is not tied to profile. It's "global" CLI config option.

Reply:
yes, removing this